src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -767,15 +875,17 @@ public void readEntityFromNBT(final NBTTagCompound compound)
         {
             updateColonyServer();
         }
-        inventory.readFromNBT(compound);
+        NBTTagList nbttaglist = compound.getTagList("Inventory", 10);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)